cloudconfig/userdatacfg_test.go

```diff
@@ -725,6 +725,56 @@ func (*cloudinitSuite) TestCloudInitConfigure(c *gc.C) {
 	}
 }
 
+func (s *cloudinitSuite) TestCloudInitConfigCloudInitUserData(c *gc.C) {
+	environConfig := minimalModelConfig(c)
+	environConfig, err := environConfig.Apply(map[string]interface{}{
+		config.CloudInitUserDataKey: validCloudInitUserData,
+	})
+	c.Assert(err, jc.ErrorIsNil)
+	instanceCfg := s.createInstanceConfig(c, environConfig)
+	cloudcfg, err := cloudinit.New("precise")
```

Comment:
Let's use "trusty" or even "xenial".
precise is past EOL now

Reply:
done